respect-lib-shared/src/commonMain/kotlin/world/respect/shared/viewmodel/report/edit/ReportEditViewModel.kt

```
@@ -0,0 +1,413 @@
+                                            )
+                                        )
+                                    )
+                                    _uiState.update { prev ->
+                                        prev.copy(
+                                            reportOptions = mockedReport
+                                        )
+                                    }
+                                } else {
+                                    _uiState.update { currentState ->
+                                        currentState.copy(
+                                            reportOptions = parsedOptions,
+                                        )
+                                    }
+                                }
+                            }
+
+                            else -> {
+                                val mockedReport = ReportOptions(
+                                    series = listOf(
+                                        ReportSeries(
+                                            reportSeriesUid = 1,
+                                            reportSeriesVisualType = ReportSeriesVisualType.BAR_CHART,
+                                            reportSeriesSubGroup = null,
+                                            reportSeriesYAxis = DefaultIndicators.list.first(),
+                                            reportSeriesFilters = emptyList()
+                                        )
+                                    )
+                                )
+                                _uiState.update { prev ->
+                                    prev.copy(
+                                        reportOptions = mockedReport
+                                    )
+                                }
+                            }
+                        }
+                    }
+                }
+            } catch (e: Exception) {
+                println("Exception $e")
```

This is hiding an exception